docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,171 @@
+This operator internally extends the functionality of `AbstractPassThruTransactionableStoreOutputOperator`. AbstractPassThruTransactionableStoreOutputOperator is the base implementation for a pass-through output adapter of a transactional store. AbstractPassThruTransactionableStoreOutputOperator implementation provides the "exactly once" guarantee. "Pass-through" means it does not wait for end window to write to the store. It will begin transaction at `beginWindow` and write to the store as the tuples come and commit the transaction at `endWindow`.
+
+**Note**: For enforcing exactly once semantics a table named 'dt_meta' must exist in the database. The sample SQL to create the same is as follows
+```
+"CREATE TABLE IF NOT EXISTS dt_meta (dt_app_id VARCHAR(100) NOT NULL, dt_operator_id INT NOT NULL, dt_window BIGINT NOT NULL, UNIQUE(dt_app_id,dt_operator_id,dt_window))".
+```
+**Note**: Additionally this operator assumes that the underlying database/table in which records are to be added supports transactions. If the database/table does not support transactions then a tuple may be inserted in a table more than once in case of auto recovery from a failure (violation of exactly once semantics).
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***0.9.4***
+3. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcTransactionableOutputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcTransactionableOutputOperator.html)***
+
+## How to Use?
+ Concrete subclasses need to choose an appropriate class (such as String or an appropriate concrete java class). Also implement a couple of abstract methods (if not using AbstractJdbcPOJOOutputOperator): `setStatementParameters(PreparedStatement statement, T tuple)` to set the parameter of the insert/update statement (which is a PreparedStatement) with values from the tuple and `getUpdateCommand()` the sql statement to update a tuple in the database. Note that these methods are also defined in AbstractJdbcPOJOOutputOperator.
+
+Several properties are available to configure the behavior of this operator and they are summarized in the table below.
+### <a name="AbstractJdbcTransactionableOutputOperatorProps"></a>Properties of AbstractJdbcTransactionableOutputOperator
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *batchSize* | Maximum number of tuples to insert in a single call (see explanation above). | int | No | 1000 |
+
+#### <a name="JdbcTransactionalStore"></a>Properties of JDBC Store (BackendStore)
```

Comment:
BackendStore => The way it is written, it looks like a Java class name but I don't think there is such a class; either omit it or provide better explanatory text, e.g. "the store which is present in .... and performs the actual JDBC operations"